contracts/fei/FSRX.sol

```diff
@@ -0,0 +1,96 @@
+import {ERC20} from "../external/solmate/tokens/ERC20.sol";
+import {ERC4626} from "../external/solmate/mixins/ERC4626.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {Constants} from "../Constants.sol";
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+library FSRXConfig {
```

Comment:
Not sure the extra code of breaking out these constants into a library is worthwhile/makes things clearer. I'd just put inside the scope of the contract as constants

Reply:
I think it's much cleaner tbh

Reply:
why not use CoreRef?